packages/utils/types/ibc.d.ts

```diff
@@ -7,40 +7,55 @@ export declare const getIbcInfo: (ibc: IBCInfo[], chain: string, counterparty: s
 export declare const getTransferChannel: (info: IBCInfo) => {
     chain_1: {
         channel_id: string;
+        client_id?: string;
```

Comment:
Why some of the .d.ts files changed without .ts files change?

Reply:
I'll check why